news/2020-02-24-mappers-report-february.md

```diff
@@ -0,0 +1,134 @@
+
+We'd like to use this section as a reminder that you can apply as a Beatmap Nominator at any time over at [BN/NAT website](https://bn.mappersguild.com/) as long as you reach the activity requirements. We hope to see some of your names on this list next month!
+
+### BN Retirements
+
+Next, we have those who have taken their leave from the group. We'd like to thank them for their tremendous work and effort within the BNs and wish them the best with their newly obtained freedom!
+
+- **osu!:** [Trynna](https://osu.ppy.sh/users/2652951)
+- **osu!taiko:** [qoot8123](https://osu.ppy.sh/users/766371)
+- **osu!catch:** [Sanyi](https://osu.ppy.sh/users/7496029), [Du5t](https://osu.ppy.sh/users/6053071) & [Rocma](https://osu.ppy.sh/users/566276)
+- **osu!mania:** [Tailsdk](https://osu.ppy.sh/users/6751666), [Critical_Star](https://osu.ppy.sh/users/3793196), [Kyousukee](https://osu.ppy.sh/users/8842107) & [MEGAtive](https://osu.ppy.sh/users/3094101)
+
+### NAT Additions
+
+In addition to the recently added BNs, some new NAT members have been brought on!
+
+The first addition we got on the 6h January was **[yaspo](https://osu.ppy.sh/users/4945926)** who served as a Beatmap Nominator for roughly over a year, he will now support the NAT for the osu! game mode. He will mainly help with BN evaluations and applications. He has also shown his skills in leading the NAT meetings already! Welcome aboard, yaspo!
+
+Our second addition we got on the 26th January is **[Greaper](https://osu.ppy.sh/users/2369776)**! With being previously a Beatmap Nominator for almost 4 months, he will now support the NAT for the osu!catch game mode. He will mainly focus on doing BN evaluations and applications and make sure, the Beatmap Nominator group will grow properly for osu!catch! So welcome aboard Greaper!
+
+## <a id="mapping-contests"></a>Mapping Contests
+
+If you are organizing a mapping contest that will begin within the next 1-3 months, feel free to submit it through [this form](https://docs.google.com/forms/d/e/1FAIpQLSeVPi2KkcWiEsJzsEs1ir79HrP2jE_Sy2DCX_4nR_EECFTX4Q/viewform?usp=sf_link) for a chance of being featured on next month's installment of the Mappers' Report. We are looking to highlight more mapping contests from the community!
+
+### Monthly Beatmapping Contest: February 2020
+
+This month's installment of the Monthly Beatmapping Contest (MBC) has arrived! Head on over to the [contest listing](https://osu.ppy.sh/home/news/2020-02-03-monthly-beatmapping-contest-february) to find out all about this month's restriction on your entries. If it's too late for you to make an entry given the deadline is in less than a week worry not, at the start of March a new contest will begin so you'll have your chance to participate. Keep your eyes peeled for the newspost which will be made when the March contest is ready.
```

Comment:
```suggestion
This month's installment of the Monthly Beatmapping Contest (MBC) has arrived! Head on over to the [contest listing](https://osu.ppy.sh/home/news/2020-02-03-monthly-beatmapping-contest-february) to find out all about this month's restriction on your entries. If it's too late for you to make an entry given the deadline is in less than a week, worry not, at the start of March a new contest will begin so you'll have your chance to participate. Keep your eyes peeled for the news post that will be made when the March contest is ready.
```

Reply:
technically you're referring to the newspost here and not the contest listing, if we're being nitpicky

Reply:
also just noticed the deadline has passed, should be reworded a bit